src/opm/io/eclipse/EclOutput.cpp

```diff
@@ -50,15 +51,79 @@ template<>
 void EclOutput::write<std::string>(const std::string& name,
                                    const std::vector<std::string>& data)
 {
+    // array type will be assumed CHAR if maximum string length is 8 or less
+    // If maximum string length is > 8, C0nn will be used with element size equal to
+    // maximum string length
```

Comment:
I guess you should check / protect against horrible large values?

Reply:
Do you agree that we stop with element size > 99 ? Especially for formatted file format longer string lengths will be very practical.

adding this to both ```write<std::string>``` functions

```c++
    if (maximum_length > 99)
        OPM_THROW(std::runtime_error, "String length > 99 not supported");
```

Reply:
> Do you agree that we stop with element size > 99 

Sure - but I guess the special value `99` should be assembled with the other constants.